beancount_n26/__init__.py

```diff
@@ -117,6 +116,20 @@ class InvalidFormatError(Exception):
 PayeePattern = namedtuple("PayeePattern", ["regex", "account"])
 
 
+class LastLineReader:
+
+    def __init__(self, fid):
+        self.fid = fid
+    def __iter__(self):
+        return self
+    def __next__(self):
+        line = self.fid.readline()
+        if len(line) == 0:
+            raise StopIteration()
```

Comment:
Does this mean that if there are empty lines in the CSV, the importer will exit early without processing the remaining lines?

Reply:
readline doesn't strip newlines, so empty lines are returned as `"\n"`
an empty string is returned on EOF